buildozer/targets/android.py

```diff
@@ -1225,4 +1225,5 @@ def cmd_logcat(self, *args):
 
 
 def get_target(buildozer):
+    buildozer.targetname = "android"
```

Comment:
I would say it's pretty bad practice and misleading mutating a get parameter, but this is how it was handled in the past